src/framework/components/anim/component-binder.js

```diff
@@ -240,10 +253,32 @@ class AnimComponentBinder extends DefaultAnimBinder {
             return new AnimTarget(function (values) {
                 setter(values);
                 propertyComponent.material.update();
-            }, animDataType, animDataComponents);
+            }, animDataType, animDataComponents, targetPath);
         }
 
-        return new AnimTarget(setter, animDataType, animDataComponents);
+        return new AnimTarget(setter, animDataType, animDataComponents, targetPath);
+    }
+
+    rebind() {
+        this.targetCache = {};
+        // #ifdef DEBUG
+        this.visitedFallbackGraphPaths = {};
+        // #endif
+
+        if (this.animComponent.entity.model?.model?.graph) {
+            this.graph = this.animComponent.entity.model?.model?.graph;
```

Comment:
just double checking that this works for render component as well .. is this.graph set to this.entity or similar somewhere?

Reply:
Yep, if a model component isn't present it'll always use the current entity as the root of the graph.